lib/bungie_client.dart

```diff
@@ -274,16 +274,16 @@ class BungieClient {
             .map((skull) => skull['displayName']));
   }
 
-  /// Returns the progress on Moments of Triumph as a completion percentage for
+  /// Returns the progress on Age of Triumph as a completion percentage for
```

Comment:
Instead of altering the existing method, add a new one specifically for Age of Triumphs and share the same code under the hood. Please also add documentation to this method to make it clear it's about Y2 MoT.